doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -131,6 +131,7 @@ nRF5340 Audio
 * Updated:
 
   * LE Audio Controller Subsystem for nRF53 from version 3303 to version 3307.
+  * Removed support for nRF5340_audio_dk boards with version 0.7.1 or older.
```

Comment:
```suggestion
  * Removed support for the nRF5340 Audio DK (PCA10121) board version 0.7.1 or older.
```
Because we're talking about board version, not a version of something on the board.